### DISCUSSION TITLE
Share vacuum.do_rooms please

### DISCUSSION THREAD

**Discussion by mdrobny** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#157] [Q&A] [answered]
Hi @PiotrMachowski!

Awesome v2 release 🎉 

In the [demo config](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/demo_config.yaml#L174) you use `vacuum.do_rooms` script for `selection_type: ROOM`.

Could you share this script please? 😄 
I configured all my rooms like in the demo and in the end I noticed this script is not part of vacuum integration 😅

**Comment by PiotrMachowski** [comment, marked as answer]
This isn't a script, that is just some fake service call to show how to use this config. Rooms should work out of the box using `vacuum_clean_segment` template
